tests/test_managers/test_tree_manager.py

```diff
@@ -55,6 +59,17 @@ def test_matches_pending_hash(tree_manager):
     assert not tree_manager.matches_pending_hash(random_hash)
 
 
+@pytest.mark.parametrize(
+    "tree_manager",
+    [Network.Ethereum],
+    indirect=True,
+)
+def test_build_function_and_send(tree_manager, mocker):
```

Comment:
What exactly does this test? Maybe check that some mocked functions were called?